src/shapes/frostedGrid.js

```diff
@@ -0,0 +1,41 @@
+module.exports = function drawFrostedGrid(p5, anchors, circles, spacing) {
```

Comment:
Similarly, a comment on each of these explaining that they are duplicates, and where they are copied from, could be helpful for future visitors.